home/home_2.1/HelloWorld.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <sstream>
+
+const std::string FILENAME = "user_stats.txt";
```

Comment:
constexpt auto
and you can use anonymous namespace to enclose this constant and decrease visibility